source/assets/js/helpers/mediaqueries.js

```diff
@@ -17,10 +17,15 @@
  * }
  *
  * // Check the current viewport against a specific breakpoint:
- * if (parseInt(estatico.mq.currentBreakpoint.value) > parseInt(estatico.mq.breakpoints.small)) {
+ * if (estatico.mq.query({ from: 'small' })) {
  * 	this.destroySmall();
  * 	this.initLarge();
  * }
+ * // or
+ * if (estatico.mq.query({ from: 'small', to: 'medium' })) {
```

Comment:
Could you update https://github.com/unic/estatico/blame/develop/docs/Coding_Guidelines.md#L120 and https://github.com/unic/estatico/blob/develop/source/demo/modules/slideshow/slideshow.js#L185 as well?

![image](https://cloud.githubusercontent.com/assets/654171/10863207/0dc31d30-7fc6-11e5-969e-09d8a8224671.png)